src/mixins/ShareMixin.js

```diff
@@ -62,6 +62,10 @@ export default {
       default() {
         return {}
       }
+    },
+    sortMethod: {
```

Comment:
sortMethod is not  related to vue-data-tables-server. so this should be added in `DataTables.vue`

Reply:
Thanks.